### PR TITLE
Refactoring: Extracted the 'Use only one x' message

### PR DESCRIPTION
**Pull request by Infernio**
Was used in two places, the EBT / SkyBirds PR will make that four.
Also added the translations from LE.

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -72,6 +72,23 @@ common:
         text: '%1%はインストールされていますが、SKSE64が見つかりません！SKSEのダウンロードページを確認してください。[SKSE64](http://skse.silverlock.org)'
       - lang: de
         text: 'Sie haben %1% installiert, aber SKSE64 wurde nicht gefunden! SKSE64 kann hier heruntergeladen werden: [SKSE64](http://skse.silverlock.org).'
+  - &useOnlyOneX
```
Comment (MacSplody) [inline masterlist.yaml]: Changes are fine. Could you move `useOnlyOneX` below the `AlreadyInX` entry?

**Review by MacSplody — CHANGES_REQUESTED** [review]
Minor position change

**Comment by Infernio** [comment]
Done.

**Review by MacSplody — APPROVED** [review]
Okay looks good. Thanks for the contribution.

**Pull request merged**